apps/web/src/types/toolResults.ts

```diff
@@ -0,0 +1,223 @@
+/**
+ * Type definitions for tool results
+ * 
+ * These interfaces define the shape of results returned by various AI tools
+ * to ensure type safety in the UI components.
+ */
+
+// Math checking tool results
+export interface MathError {
```

Comment:
I'm  a bit curious if these types can be in the tools code of /ai/ dir, then imported here. Web does import from /ai/.